architecture/rhcos.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+[id='rhcos']
+= Red Hat CoreOS in {product-title}
```

Comment:
@chrisnegus, here's the architecture stub for RHCOS.

Reply:
Thanks Kathryn!